pmd-scala-modules/pmd-scala_2.12/pom.xml

```diff
@@ -22,6 +22,42 @@
                 <groupId>org.jetbrains.kotlin</groupId>
             </plugin>
         </plugins>
+        <pluginManagement>
+        	<plugins>
+        		<!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
+        		<plugin>
+        			<groupId>org.eclipse.m2e</groupId>
+        			<artifactId>lifecycle-mapping</artifactId>
+        			<version>1.0.0</version>
+        			<configuration>
+        				<lifecycleMappingMetadata>
+        					<pluginExecutions>
+        						<pluginExecution>
+        							<pluginExecutionFilter>
+        								<groupId>
+        									org.jetbrains.kotlin
+        								</groupId>
+        								<artifactId>
+        									kotlin-maven-plugin
+        								</artifactId>
+        								<versionRange>
+        									[1.7.20,)
+        								</versionRange>
+        								<goals>
+        									<goal>compile</goal>
+        									<goal>test-compile</goal>
+        								</goals>
+        							</pluginExecutionFilter>
+        							<action>
+        								<ignore></ignore>
+        							</action>
+        						</pluginExecution>
+        					</pluginExecutions>
+        				</lifecycleMappingMetadata>
+        			</configuration>
+        		</plugin>
+        	</plugins>
+        </pluginManagement>
```

Comment:
Accepted, of course. The maven configuration file was changed because of the new package reference. No need to explain too much.